rotkehlchen/api/v1/resources.py

```diff
@@ -2081,3 +2083,19 @@ def post(self, file: FileStorage) -> Response:
             file.save(str(filepath))
             response = self.rest_api.import_user_assets(path=filepath)
         return response
+
+
+class DBSnapshotExportingResource(BaseResource):
+    get_schema = SnapshotExportingSchema()
+
+    @use_kwargs(get_schema, location='json_and_query')
```

Comment:
I would remove the query params. I realize we have it in other non-get endpoints, though so we can do it in a different PR.